dac/backend/src/main/java/com/dremio/dac/api/JobResource.java

```diff
@@ -81,7 +81,8 @@ public JobStatus getJobStatus(@PathParam("id") String id) {
   @GET
   @Path("/{id}/results")
   public JobData getQueryResults(@PathParam("id") String id, @QueryParam("offset") @DefaultValue("0") Integer offset, @Valid @QueryParam("limit") @DefaultValue("100") Integer limit) {
-    Preconditions.checkArgument(limit <= 500,"limit can not exceed 500 rows");
+    long queryResultsLimit = 5000;
```

Comment:
Preferable to add this do the dremio.conf, leave the default at 500, and allow for adjustment?

Reply:
Sure, it was an initial idea, but I didnt find any *Resource class refers to DremioConfig and thought it is a pattern. Will refer to DremioConfig thru injectable SabotContext

Reply:
> Preferable to add this do the dremio.conf, leave the default at 500, and allow for adjustment?

@patricker I'm sorry to bother you in this way，I want to know how the dremio-ce*.jar source code can be obtained? (I've been reading [Is Dremio CE fully Apache 2.0?](https://community.dremio.com/t/is-dremio-ce-fully-apache-2-0/3858))

Reply:
For research purposes, you could decompile the code (JD Gui), or view the auto-decompiled class files in IntelliJ.  Please keep in mind that you DO NOT have a license to use this source code :(

I dug pretty deep into the CE-JDBC code recently, but was not able to resolve the issue myself because of licensing constraints, ref: https://community.dremio.com/t/enhancement-request-arp-jdbc/4449/.

Reply:
My email address is available under my profile, if you need to email me instead of discussing here: https://github.com/patricker